frontend/packages/console-shared/src/hooks/__tests__/usePluginsOverviewTabSection.spec.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { useExtensions } from '@console/plugin-sdk';
+import {
+  knativeServiceObj,
+  sampleKnativeRevisions,
+} from '@console/knative-plugin/src/topology/__tests__/topology-knative-test-data';
+import { OverviewItem } from '../../types/resource';
+import { testHook } from '../../test-utils/hooks-utils';
+import { usePluginsOverviewTabSection } from '../plugins-overview-tab-section';
+
+jest.mock('@console/plugin-sdk/src/useExtensions', () => ({
+  useExtensions: jest.fn(),
+}));
+describe('useSearchFilter', () => {
+  let item: OverviewItem;
+  beforeEach(() => {
+    item = {
+      revisions: sampleKnativeRevisions.data,
+      obj: knativeServiceObj,
+      buildConfigs: [],
+    } as OverviewItem;
+  });
+
+  it('should be empty, if there is no overview tab section registered', () => {
+    testHook(() => {
+      (useExtensions as jest.Mock).mockReturnValue([]);
+      expect(usePluginsOverviewTabSection(item)).toHaveLength(0);
+    });
+  });
+  it('should not be empty, when there is overview tab section registered', () => {
+    testHook(() => {
+      (useExtensions as jest.Mock).mockReturnValue([
+        {
+          properties: {
+            key: 'revisions',
+            loader: React.createElement(Button, { children: 'test' }),
+          },
+        },
```

Comment:
I suggest at least typing this value to the matching extension type.

Reply:
added type.